src/Storage.h

```diff
@@ -1,7 +1,7 @@
 /**
  * @file Storage.h
  *
- * @brief
+ * @brief Provides functions to manage data in a non-volatile memory
```

Comment:
```suggestion
 * @brief Provides functions to manage data in non-volatile memory
```

Reply:
I just copy-pasted this from the implementation file but I'll change it in both files.